### PR TITLE
New page added for emoji skin tones

### PR DESCRIPTION
**Pull request by cwarnermm**

### CODE REVIEW

File: source/help/messaging/emoji-skin-tones.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Emoji Skin Tones
+================
+
+Emoji are pictures that can communicate emotions, gestures, intent, and so much more. The Mattermost Emoji Picker uses neutral skin tones by default. You can customize the skin tone for many emoji using a native emoji picker.
+
+Selecting Emoji
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:6]:
```suggestion
Selecting Emojis
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+---------------
+
+Click the smile icon inside the message input box to open the Mattermost Emoji Picker.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:9]:
```suggestion
Select the smile icon inside the Mattermost message input box to open the Emoji Picker.
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+2. Select an alternate skin tone and then select an emoji.
+
+.. image:: ../../images/windows10emojiskintones.png
+  :alt: Select alternate emoji skin tones on Windows 10 systems.
+  :scale: 70
+
+**MacOS**
+
+1. Use the keyboard shortcut CMD+CTRL+Space Bar to open the native emoji picker for MacOS.
+2. Click and hold on an emoji to select an alternate skin tone.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:31]:
```suggestion
2. Select and hold on an emoji to select an alternate skin tone.
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+.. image:: ../../images/macosemojiskintones.png
+  :alt: Select alternate emoji skin tones on MacOS systems.
+  :scale: 80
+
+**iPhone and iPad**
+
+1. Open the iPhone Emoji Keyboard.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:39]:
```suggestion
1. Open the emoji keyboard.
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  :alt: Select alternate emoji skin tones on Apple devices.
+  :scale: 30
+
+
+.. tip::
+  If you don't see the emoji keyboard for your Apple device, you can enable it. See `Use emoji on your iPhone, iPad, and iPod touch <https://support.apple.com/en-us/HT202332>`__ for details.
+
+**Android**
+
+1. Open an Android Emoji Keyboard.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:52]:
```suggestion
1. Open an Android emoji keyboard.
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+**Windows 10**
+
+1. Use the keyboard shortcut: Windows Key + . (period) to open the native emoji picker for Windows 10.
+2. Select an alternate skin tone and then select an emoji.
+
+.. image:: ../../images/windows10emojiskintones.png
+  :alt: Select alternate emoji skin tones on Windows 10 systems.
+  :scale: 70
+
+**MacOS**
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:28]:
```suggestion
**macOS**
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+1. Use the keyboard shortcut CMD+CTRL+Space Bar to open the native emoji picker for MacOS.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:30]:
```suggestion
1. Use the keyboard shortcut CMD+CTRL+Space Bar to open the native emoji picker for macOS.
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+  :alt: Open the Mattermost Emoji Picker.
+
+Selecting Emoji Skin Tones
+--------------------------
+
+To access alternate skin tones, use the native emoji pickers available on your platform or device.
+
+**Windows 10**
+
+1. Use the keyboard shortcut: Windows Key + . (period) to open the native emoji picker for Windows 10.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:21]:
```suggestion
1. Use the keyboard shortcut: Windows Key+. (period) to open the native emoji picker for Windows 10.
```

File: source/help/messaging/emoji-skin-tones.rst [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+Click the smile icon inside the message input box to open the Mattermost Emoji Picker.
+
+.. image:: ../../images/selectemoji.png
+  :alt: Open the Mattermost Emoji Picker.
+
+Selecting Emoji Skin Tones
+--------------------------
+
+To access alternate skin tones, use the native emoji pickers available on your platform or device.
```
Comment (justinegeffen) [inline source/help/messaging/emoji-skin-tones.rst:17]:
```suggestion
To access alternate skin tones, open Mattermost on your device and use the native emoji pickers provided on your platform or device.
```

Comment (justinegeffen) [reply source/help/messaging/emoji-skin-tones.rst:17]: For the steps below, does the user need to be in Mattermost first? Not sure if we need to state that for all the sections or whether my suggestion will suffice? 

Also, not sure if it's something we should note but this only applies to "hand" emojis (not smileys etc) and the selection will remain as such going forward until/unless changed.

**Comment by justinegeffen** [comment]
Thanks @cwarnermm! This is really helpful. I added some comments and suggestions. I noted that although the images aren't rendering when I preview the content in this PR, they are rendering when I open the file directly. I've had this problem before - not sure what causes it but just FYI that they do seem to be working.

**Comment by esethna** [comment]
Replaced with #4135

**Pull request closed without merging**